tests/test_run.py

```diff
@@ -266,7 +266,9 @@ def only_eynollah(logrec):
     for options in [
             [], # defaults
             ["-doit", str(outrenderfile.parent)],
-            ["-trocr"],
+            # Disabled until resource consumption / GPU OOM / "The Conv2D op currently only supports the NHWC tensor format on the CPU. The op was given the format: NCHW" error fixed
+            # kba Fri Sep 26 12:33:16 CEST 2025
+            # ["-trocr"],
```

Comment:
It was **not** the -trocr that failed, but the TF-based Transformer OCR, i.e. the other two options!

Reply:
For me locally, the other way around. Confusing. I've removed that commit and force-pushed one doing the opposite (disabling default and `-doit` and keeping `-trocr`).